.github/workflows/build.yml

```diff
@@ -281,6 +281,47 @@ jobs:
             ../scripts/xcodebuild.sh macos/TemplateExample.xcworkspace build
           fi
         working-directory: template-example
+  windows:
+    name: "Windows"
+    runs-on: windows-latest
+    strategy: 
+      matrix:
+        configuration: [Debug, Release]
+        platform: [ARM, x64, x86]
+    steps:
+      - name: Set up MSBuild
+        uses: microsoft/setup-msbuild@v1.0.1
+      - name: Setup VSTest.console.exe
+        uses: darenm/Setup-VSTest@v1
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install
+        run: |
+          yarn
+        working-directory: example
+      - name: Build bundle and create solution
+        run: |
+          yarn build:windows
```

Comment:
We should move `yarn build:windows` to the "Build" step.

Reply:
If we move `yarn build:windows` to "Build" step and solution generation to install, then bundle assets will not be included in the project. It doesn't really matter in terms of CI right now though.

Reply:
Ah, you're right.